libvast/vast/concept/printable/detail/as_printer.hpp

```diff
@@ -48,6 +48,8 @@ constexpr auto as_printer(T x) -> std::enable_if_t<is_printer_v<T>, T> {
 
 // -- binary ------------------------------------------------------------------
 
+// clang-format off
```

Comment:
Can you move this to around line 90?

Reply:
That will destroy the formatting of the existing code though, which looks like hand-crafted and which is currently not liked by `clang-format`.

Reply:
Alright, in that case go ahead without this change.